apps/web/playwright/event-types.e2e.ts

```diff
@@ -142,17 +142,25 @@ test.describe("Event Types tests", () => {
         .first()
         .getAttribute("href");
 
+      /**
+       * Verify first organizer address
+       */
       await page.goto(previewLink ?? "");
-
       await selectFirstAvailableTimeSlotNextMonth(page);
-
-      for (const location of locationData) {
-        await page.locator(`span:has-text("${location}")`).click();
```

Comment:
The test was selecting all locations one by one and thus in the end only last location was selected which would work fine.

Now we are testing selection of two different locations one by one